src/pages/config.tsx

```diff
@@ -249,8 +249,20 @@ const ConfigPage: FunctionComponent<ConfigPageProps> = () => {
           />
         </InputSection>
         <div className="w-100 inline-flex flex-row justify-between">
-          <button className="e2e-config-page-button button-reset mr5 pv3 tc bg-animate hover-bg-gold pointer w-30 bn" id="reset-config" onClick={() => { void doResetConfig() }}>Reset Config</button>
-          <ServiceWorkerReadyButton className="e2e-config-page-button white w-100 pa3" id="save-config" label={isSaving ? 'Saving...' : 'Save Config'} waitingLabel='Waiting for service worker registration...' onClick={() => { void saveConfig() }} />
+          <ServiceWorkerReadyButton
+            className="e2e-config-page-button pv3 tc bg-animate hover-bg-red-muted pointer w-30 bn"
+            id="deregister-sw"
+            label="Deregister SW"
+            waitingLabel='Waiting for SW...'
```

Comment:
nit: "Reset Worker" will look better next to "Reset Config"

```suggestion
            label="Reset Worker"
            waitingLabel='Waiting for SW...'
```